AttributeError for `data` on `Submission`s

```
_______________________________________________________________ test_reply_content_downloaded ________________________________________________________________

mocker = <pytest_mock.MockFixture object at 0x7fef8f32ec50>

    def test_reply_content_downloaded(mocker):
        source = factory.Source()
        journalist = User('Testy mcTestface')
        reply = Reply(source=source, uuid="test", size=123,
                      filename="test.docx", journalist=journalist)
        reply.is_downloaded = True
        mocker.patch('builtins.open', mocker.mock_open(read_data="blah"))
>       assert reply.content == "blah"

tests/test_models.py:56: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <Reply test.docx>

    @property
    def content(self):
        if self.is_downloaded:
            fn_no_ext, _ = os.path.splitext(self.filename)
>           return self.data.get(fn_no_ext)
E           AttributeError: 'Reply' object has no attribute 'data'

securedrop_client/models.py:33: AttributeError
_____________________________________________________________ test_submission_content_downloaded _____________________________________________________________

mocker = <pytest_mock.MockFixture object at 0x7fef8f31d358>

    def test_submission_content_downloaded(mocker):
        source = factory.Source()
        submission = Submission(source=source, uuid="test", size=123,
                                filename="test.docx",
                                download_url='http://test/test')
        submission.is_downloaded = True
        mocker.patch('builtins.open', mocker.mock_open(read_data="blah"))
>       assert submission.content == "blah"

tests/test_models.py:38: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <Submission test.docx>

    @property
    def content(self):
        if self.is_downloaded:
            fn_no_ext, _ = os.path.splitext(self.filename)
>           return self.data.get(fn_no_ext)
E           AttributeError: 'Submission' object has no attribute 'data'

securedrop_client/models.py:33: AttributeError
```

This was uncovered while repeatedly running the tests in #196 (randomized ordering).

This is directly related to #197.

--- a/tests/test_models.py
+++ b/tests/test_models.py
@@ -1,4 +1,5 @@
 from tests import factory
+from securedrop_client.data import Data
 from securedrop_client.models import Reply, Submission, User
 
 
@@ -33,6 +34,7 @@ def test_submission_content_downloaded(mocker):
     submission = Submission(source=source, uuid="test", size=123,
                             filename="test.docx",
                             download_url='http://test/test')
+    submission.data = Data('foo')
     submission.is_downloaded = True
     mocker.patch('builtins.open', mocker.mock_open(read_data="blah"))
     assert submission.content == "blah"
@@ -51,6 +53,7 @@ def test_reply_content_downloaded(mocker):
     journalist = User('Testy mcTestface')
     reply = Reply(source=source, uuid="test", size=123,
                   filename="test.docx", journalist=journalist)
+    reply.data = Data('foo')
     reply.is_downloaded = True
     mocker.patch('builtins.open', mocker.mock_open(read_data="blah"))
     assert reply.content == "blah"

